lib/utils.js

```diff
@@ -29,6 +29,11 @@ function configureBinaryLog (opts) {
     let l = this.levels[level];
     if (l < this.levels[this.level]) return; // eslint-disable-line curly
     actualLog(level, prefix, msg);
+
+    // TODO: Callback function to handle logs
+    if (opts.onLogMessage) {
```

Comment:
if (_.isFunction(opts.onLogMessage))

Reply:
Uhm🤔
OK, I will update this, wait a second